examples/determinator-rails/config/initializers/determinator.rb

```diff
@@ -1,7 +1,7 @@
-require 'determinator/retrieve/routemaster'
+require 'determinator/retrieve/dynaconf'
 require 'active_support/cache'
 
-retrieval = Determinator::Retrieve::Routemaster.new(discovery_url: 'https://flo.dev/')
+retrieval = Determinator::Retrieve::Dynaconf.new(host: 'localhost:2345')
```

Comment:
Would it make sense to allow (as an option, at least) falling back on the service if the local DynaConf is misbehaving?

Reply:
I was going to suggest this - we have something similar in our implementation [here](https://github.com/deliveroo/rs-accounts/pull/254/files#diff-b16380e1faa257e2f16f06e72b7fd1e0R35)

Reply:
Not sure what you mean, we have a DynaConf also running centrally and available as a service?

Reply:
That's right. The service is always available at `https://dynaconf.${HOPPER_DOMAIN_NAME}`, where `$HOPPER_DOMAIN_NAME` is injected at runtime by `hopper-runner`.

Reply:
The service is just all the local daemons, behind a load balancer.

Reply:
I'm personally not convinced that's a good idea considering this tends to be used very much in the hotpath. With some cache of course but still with a chance of hitting the remote server. I'm also not strictly against it though.

Can we perhaps consider it for a minor version improvement as a next step? I'd like to get the first version of this released asap.

For implementing what you're suggesting, I'm thinking about allowing `base_url` to be an array and the endpoints are simply hit in order, should any of them fail. But implementing that needs a bit more testing, etc. So I'd rather keep it as an improvement, rather than a blocker.

Reply:
It's not a blocker for me. I was just thinking it's good for reliability.